src/components/Dashboard/FilingHistoryList/subtitles/SubtitleRejected.vue

```diff
@@ -2,7 +2,16 @@
   <div class="subtitle-rejected">
     <span class="orange--text text--darken-2">REJECTED</span>
     <span class="vert-pipe" />
-    <span>PAID (filed by {{ filing.submitter }} on <DateTooltip :date="submittedDate" />)</span>
+    <!-- Handle Continuation Filings -->
+    <span v-if="EnumUtilities.isTypeContinuationIn(filing)">
+      <span v-if="filing.paymentStatusCode === 'COMPLETED'">
+        PAID (submitted by {{ filing.submitter }} on <DateTooltip :date="submittedDate" />)
+      </span>
+      <span v-else>
+        Submitted by {{ filing.submitter }} on <DateTooltip :date="submittedDate" />
+      </span>
+    </span>
+    <span v-else>PAID (filed by {{ filing.submitter }} on <DateTooltip :date="submittedDate" />)</span>
```

Comment:
I think you can undo this code.

(In the old flow, users paid before the authorization files were reviewed, which could end in this situation. In the new flow, users pay after the authorization is approved, so this scenario will never happen. We don't need to handle this scenario. Instead, we should fix the data in the db or just delete it.)

Reply:
Yes, removed